app/gui/view/graph-editor/src/builtin/visualization/java_script/sql.js

```diff
@@ -81,9 +81,9 @@ loadScript('https://cdnjs.cloudflare.com/ajax/libs/sql-formatter/4.0.2/sql-forma
  */
 class SqlVisualization extends Visualization {
     // TODO Change the type below once #837 is done:
```

Comment:
unrelated but I'm not sure this TODO is relevant anymore

Reply:
Removed it